### PR TITLE
Fixes issue-189

### PR DESCRIPTION
**Pull request by tcavaletto**

### CODE REVIEW

File: app/helpers/notification_helper.rb [hunk 1]
```diff
@@ -44,6 +44,10 @@ def construct_notification_message_for(n)
       content_tag(:p,
                   "#{link_to_author(n.author)} has deleted a save on #{link_to_curriculum(n.curricula, n.author)}".html_safe,
                   class: 'bg-danger text-danger')
+    when 8
+      content_tag(:p,
+                  "#{link_to_author(n.author)} has been added as a contributor to #{link_to_curriculum(n.curricula, n.author)}".html_safe,
+                  class: 'bg-danger text-primary')
```
Comment (bammons) [inline app/helpers/notification_helper.rb]: why bg-danger? Is it a bad thing that they added a contributor?

File: config/application.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
 Bundler.require(:default, Rails.env)
 
 module GitMyCurriculum
+    # Filler Comment here
```
Comment (bammons) [inline config/application.rb]: why?

File: app/helpers/notification_helper.rb (hunk 1 above)
Comment (tcavaletto) [inline app/helpers/notification_helper.rb]: Ah. Only caught the text-danger which I corrected. New commit fixes this.

File: config/application.rb (hunk 2 above)
Comment (tcavaletto) [inline config/application.rb]: Rubocop on computer threw error requesting comments above class files for this and 30 others. Given that it passed travis tests regardless, the new commit has this deleted.

**Comment by bammons** [comment]
:+1:

**Comment by Linkster700** [comment]
+1

**Comment by kdorer** [comment]
+1

**Pull request merged**